smelli/classes.py

```diff
@@ -223,6 +250,15 @@ def _load_likelihoods(self,
                 continue
             with open(self._get_yaml_path(fn), 'r') as f:
                 yaml_dict = flavio.io.yaml.load_include(f)
+                if fn in add_measurements.keys():
+                    yaml_dict['include_measurements'] += add_measurements[fn]
```

Comment:
Same comment as above, I think it would be good to test for the presence of the added measurements.